langchain/src/document_loaders/web/confluence.ts

```diff
@@ -100,11 +100,14 @@ export class ConfluencePagesLoader extends BaseDocumentLoader {
   /**
    * Fetches all the pages in the specified space and converts each page to
    * a Document instance.
+   * @param options the extra options of the load function
+   * @param options.limit The limit parameter to overwrite the size to fetch pages.
+   * @param options.start The start parameter to set inital offset to fetch pages.
    * @returns Promise resolving to an array of Document instances.
    */
-  public async load(): Promise<Document[]> {
+  public async load(options?: { start?: number, limit?: number }): Promise<Document[]> {
```

Comment:
A little worried that we're deviating from the base interface, but this seems reasonable.

Reply:
the base interface doesn't support a few things to make this a really useful loader for indexing a full confluence space. 

Planning to add support to [expand](https://developer.atlassian.com/cloud/confluence/rest/v1/intro/#using:~:text=use%20the%20expand%20query%20parameter) parameter in my next patch as well